x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/add_prepackaged_rules_route.ts

```diff
@@ -89,17 +91,22 @@ export const createPrepackagedRules = async (
   siemClient: AppClient,
   alertsClient: AlertsClient,
   frameworkRequest: FrameworkRequest,
-  maxTimelineImportExportSize: number
+  maxTimelineImportExportSize: number,
+  exceptionsClient?: ExceptionListClient
 ): Promise<PrePackagedRulesAndTimelinesSchema | null> => {
   const clusterClient = context.core.elasticsearch.legacy.client;
   const savedObjectsClient = context.core.savedObjects.client;
+  const exceptionsListClient =
+    context.lists != null ? context.lists?.getExceptionListClient() : exceptionsClient;
```

Comment:
Since we are checking if `context.lists != null` then we should not need the optional chaining as we have asserted that the lists property is present and not null on `context`.